p2pd/main.go

```diff
@@ -71,7 +77,7 @@ func main() {
 		opts = append(opts, libp2p.NATPortMap())
 	}
 
-	d, err := p2pd.NewDaemon(context.Background(), *sock, opts...)
+	d, err := p2pd.NewDaemon(context.Background(), *maddr, opts...)
```

Comment:
why is this a pointer dereference? it should be just maddr.

Reply:
Merge failure on my part. Fixed.